bcgov/HOUS-permit-portal
# filter out of date permit applications for notification link

## Description

Refactor search filters in order to provide a link for template update notifications 

## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-1612

## Steps to QA

Update template as super admin
get notifcation as submitter
click the Go link
You should see only the relevant draft permits with yellow highlighting
reset the filter to see everything again.
